doc/source/user_guide/fields_container.rst

```diff
@@ -1,23 +1,22 @@
 .. _ref_user_guide_fields_container:
 
 ===========================
-Fields container and fields
+Field containers and fields
```

Comment:
This is a tough change that should be run past the PyDPF-Core team. the As shown below, the name of the API object is FieldsContainer:
![image](https://github.com/user-attachments/assets/e0b0c9ce-221a-4806-9cf7-7f4f73279f88)

Reply:
I know, but here they define it as "field container"
https://dpf.docs.pyansys.com/version/stable/user_guide/concepts/concepts.html
and they used the singular in a few other spots too. I guess I could change the singular to the plural, but it sounds odd!.

Reply:
@JennaPaikowsky I agree that t does sound odd. I'd still run it past the team though, given how the object is named. I definitely prefer field container, but I thought I had a discussion long ago where I was told that it should be fields container. I'm sure I'll see the term used elsewhere, so I'd like to know the outcome. Thanks!

Reply:
Replying here too: I agree this is odd, but sadly this is the kind of naming we do not really want to change. The same logic applies to several other entities.